pom.xml

```diff
@@ -498,7 +493,8 @@
                                             <banLatest>true</banLatest>
                                             <banRelease>true</banRelease>
                                             <banSnapshots>false</banSnapshots>
-                                            <phases>clean,deploy,site</phases>
+                                            <phases>clean,deploy</phases>
+                                            <unCheckedPluginList>org.apache.maven.plugins:maven-site-plugin</unCheckedPluginList>
```

Comment:
Do we actually need this after the `site` phase is removed?

Reply:
yeah it detects the maven-site-plugin even if phases is set to just `clean` or just `deploy`.

Reply:
maybe just because site is so deeply ingrained in maven https://maven.apache.org/ref/3.6.3/maven-core/lifecycles.html#site_Lifecycle it's one of the 3 predefined lifecycles